### PR TITLE
adding betking

### PR DESCRIPTION
**Pull request by Farelz**
Added Betking To The List. Let Me Know What You Think?

### CODE REVIEW

**Comment by dbkonxepts** [comment]
Hi @farelz Thanks for your contribution, check to make sure your SVG file doesn't contain any image. 

Also use the tools specified in contribution.md to optimize your SVG file.

File: logos/betking/betking.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...gsAZYoEOBZpbQo4AVWwvmmGZGLecWpkUk6CxLRTJEUXPqQuTcmcuPXWxV39n/yJ//pzP/+Y hzzsyhjHbTNs6nGnmzIQW0jLEIO5gtgpV7RTB4yChNnCjLWwFq2v6+H6csc4P7Lt5uMf/oD9u5NP fvSWN/zl3+ZsC7N1wVkJ2g2HGMPMUDqCGk8aUlNxaI0EZwNzSzFQo6YinhqaWlRp9FlQEwKFyHfS ddSdodbdBhMRjLHW7gSsewLEkrKk80RICcqREEkji1dhV6xublW+dZmNLG0MzFaDEoQpitYgTxwP HtyfOoBUmlaJOv2BEnwMMHTqVLzjSO40dHs8pwDAlu1czpx588QJ5wypgtQZDcEP+t3RaMYUZygT mM+MVphHK5C6LKvb1iapAu/9wOETq+Xesy5++1ve84evfBMM2JpWAGNAiCHoqSyN5l8YYDCBQYYY PrbgaGwEapAHtSY1XsrKT5Vj0nE2tU3btKtrCG072fLl1sru5Zf//M895tH9D3xgdM01VyepMRbG ar/b9Q0YYDtrV+npfPPUehBEECzPukvqG4ggTcxgV1dlNF2/7Wnf9PhvevLFKvjAhz85nkgn7+1Z NJeev+jEO9jgTdBEhRmGRZ0SQgPAmkJREKWqhFnWRXFOsFBjhO2OoPs9DTbGgPm0zdeOa87XAVYQ AlEDnoJLUFSCECKRshHiSHzDLTef3LqdsgnSTa8b1lqoadtgExtRRaptKk3cUlsmHY88+GY9mmlD 44muI/cyy6FOuYXAzG+6vQ03v1lgJnCeMhUANjcmu3afzbDTza0kyeuyMeDYyGI/nT0XoQBSUiY9 MwmQWfSKAGzqFaMS//N//a5xS2Xtusvn/tIvv+LQsRgAYUzbGAB2DBPBAXx6WjhrJ/FcIQdpZsAh aolYAS1im+UWpCE2xEKkPjRsTG/vyqBjBxkO7Fn89V95+fO+8/IbbsD/+B8vm1R1iJq4DMCRI0e8 h3UAAXF7VRqBcCpmBcAbiCIAoQ3TJAUbbI1ObI2On9i84ZL7nfPrv/3jMPj8zdXvv/IvOov7Vjc2 u1ncu5xbgUUekQu6MeQmpp3GDhopWimicW1ufCehHqKdteoCI7ANyALyiDTO/Xv0jrcd3H2wqkIk xhhjPBWtZtnWDu4y5plJAFXgatZgUoKyEjNgFOktNx89eWIDgD
```
Comment (dbkonxepts) [inline logos/betking/betking.svg]: Hi, @Farelz Thanks for your contribution. 
However, there seems to be an issue with the SVG file; it contains a PNG image. i.e` xlink:href="data:image/png;`
1. Do re-edit the SVG to make sure its 100% vector.
2. Go through the [**contribution.md**](https://github.com/PaystackHQ/nigerialogos/blob/master/contributing.md) on how to optimize the SVG.

**Comment by Farelz** [comment]
Thanks for the reply. Will try and get it sorted out. Do have a nice
evening.

**Comment by fathermerry** [comment]
Heey @Farelz! Thanks for sending this in. Can you kindly review the comments on your PR? We'll like to either review or close this by the end of this week.

**Pull request closed without merging**